app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/CollectionHeaderViewHolder.kt

```diff
@@ -4,14 +4,66 @@
 
 package org.mozilla.fenix.home.sessioncontrol.viewholders
 
-import android.view.View
-import androidx.recyclerview.widget.RecyclerView
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.LifecycleOwner
 import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.ComposeViewHolder
+import org.mozilla.fenix.compose.SectionHeader
+import org.mozilla.fenix.theme.FirefoxTheme
 
 class CollectionHeaderViewHolder(
-    view: View
-) : RecyclerView.ViewHolder(view) {
+    composeView: ComposeView,
+    viewLifecycleOwner: LifecycleOwner,
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
     companion object {
         const val LAYOUT_ID = R.layout.collection_header
     }
+
+    @Composable
+    override fun Content() {
+        Column {
+            Spacer(modifier = Modifier.height(40.dp))
+
+            CollectionHeader(
+                headerText = stringResource(R.string.collections_header),
+            )
+
+            Spacer(Modifier.height(10.dp))
+        }
+    }
+}
+
+@Composable
+fun CollectionHeader(
+    headerText: String
+) {
+    FirefoxTheme {
+        SectionHeader(
+            text = headerText,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(16.dp, 16.dp, 16.dp, 0.dp)
```

Comment:
this could probably be simplified by making it `.padding(horizontal = 16.dp)` and adding the 16 top padding to the top Spacer.

Reply:
Can you add the init {} with the horizontal margins and I think that will avoid needing to do `.padding(horizontal = 16.dp)`

Reply:
I will need to look at the designs and think about this 16 top padding, but we should probably just bump the 40.dp spacer to 56 dp. I want to look at some designs to see if we have the expected spacing